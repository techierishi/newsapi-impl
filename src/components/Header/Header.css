header {
    background-color: #171717;
    padding: 8px 0;
    -webkit-box-shadow: 0px 5px 10px #4e4e4e;
    -moz-box-shadow: 0px 5px 10px #4e4e4e;
    box-shadow: 0px 5px 10px #4e4e4e;
}

header p {
    font-size: 30px;
    line-height: 38px;
    color: #fff;
    font-weight: 400;
    margin-bottom: 0;
    text-align: center;
}