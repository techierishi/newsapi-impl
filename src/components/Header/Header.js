import React from "react";
import { Row, Col } from "shards-react";

import "./Header.css";

export default class Header extends React.Component {
  render() {
    return (
      <header>
        <Row>
          <Col >
             <p>News</p>
          </Col>
        </Row>
      </header>
    );
  }
}
