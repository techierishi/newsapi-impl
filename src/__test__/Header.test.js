// Link.react.test.js
import React from "react";
import renderer from "react-test-renderer";
import Header from "../components/Header/Header";

test("Header snapshot test", () => {
  const component = renderer.create(<Header />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
