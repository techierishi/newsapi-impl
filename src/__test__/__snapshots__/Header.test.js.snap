// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot test 1`] = `
<header>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <p>
        News
      </p>
    </div>
  </div>
</header>
`;
